compute/instances/quickstart.mdx

```diff
@@ -23,34 +23,33 @@ Scaleway [Instances](/compute/instances/concepts/#instance) are computing units
+    2. Click **Create Instance**. The [Instance creation page](https://console.scaleway.com/instance/servers) displays.
+    3. Complete the following steps:
+        - **Choose an Availability Zone**, which represents the geographical region where your Instance will be deployed.
+        - **Choose an Instance type**. <br />
+          Instance offers vary in pricing, processing power, memory, storage, and bandwidth. [Discover the best Instance type for your needs](/compute/instances/reference-content/choosing-instance-type/).
+        - ***Choose an Image** to run on your Instance. <br />
+          This can be an operating system, an InstantApp, or a custom image. [Check all available Linux distributions and InstantApps](/compute/instances/reference-content/images-and-instantapps/).
+        - **Name** your Instance, or leave the randomly-generated name in place. Optionally, you can add [tags](/compute/instances/concepts/#tags) to help you organize your Instance.
+        - **Add Volumes**, which are storage spaces used by your Instances. A block volume with a default name and 5,000 IOPS is automatically provided for your system volume. You can customize this volume and attach up up to 16 local and/or block type volumes as needed.
+          <Message type="important">
+             - Ensure that the volume containing your OS image has a minimum size of 10 GB. For a GPU OS, the recommended size is 125 GB.
+             - When multiple Block Storage volumes are linked to your Instance, the primary volume will host the OS and is essential for booting the Instance. Once the Instance is created can [modify your boot volume](/compute/instances/how-to/use-boot-modes/#how-to-change-the-boot-volume).
+             - Booting from a volume that either lacks an OS or is among multiple volumes with identical operating systems can lead to inconsistent boot outcomes.
+          </Message>
+        - **Configure network** of the Instance.
+          - Leave the checkbox ticked to assign a **Public IPv4** to the Instance. You can either allocate a new IPv4 address or select one or multiple existing IPv4s. Alternatively, uncheck the box if you do not want an IPv4.
+          - Leave the checkbox ticked to assign a **Public IPv6** to the Instance. You can either allocate a new IPv6 address or select one or multiple existing IPv6s. Alternatively, uncheck the box if you do not want an IPv6.
+            <Message type="tip">
+              You can attach up to 5 IPs to an Instance, combining IPv4 and IPv6 addresses, which is useful for running different services or applications on the same Instance.
+            </Message>
+        - (Optional) Click **Advanced options** if you want to configure a [cloud-init configuration](/compute/instances/concepts/#cloud-init). Otherwise, leave these options at their default values.
```

Comment:
```suggestion
        - (Optional) Click **Advanced options** to configure a [cloud-init configuration](/compute/instances/concepts/#cloud-init). Otherwise, leave these options at their default values.
```